pkg/models/devops/devops_test.go

```diff
@@ -128,3 +132,40 @@ func TestGetBranchNodesDetail(t *testing.T) {
 		}
 	}
 }
+
+func TestConvertToHttpParameters(t *testing.T) {
```

Comment:
I didn't find any code lines against the following error:

`fatal error: concurrent map writes`

Reply:
please see <https://github.com/golang/go/issues/35317>.

Reply:
what I want to say is that I cannot find any test cases against the error from this test function.